src/_component/about/about.pug

```diff
@@ -0,0 +1,33 @@
+.column(span="row")
+  .section__header-wrapper
+    .section__header=data['about'].sectionHeader
```

Comment:
Nothing wrong with your  `data['about'].sectionHeader` but you could have had just `data.about.sectionHeader` as you don't have any special character like `-` in the name of your file about.json